### PR TITLE
Added ex. doc to (nyxt-init-file) function+doc for devel sly testing

### PR DESCRIPTION
**Pull request by opalvault**
Getting feet wet here. Just adding documentation to dev manual + a function in start.lisp.

### CODE REVIEW

File: documents/README.org [hunk 1]
```diff
@@ -206,8 +205,7 @@ Notes:
    `webkit2-gtk(-devel)` instead of `webkit-gtk3` via your favorite
    package manager. If you are on a legacy version of macOS which is not
    supported by HomeBrew, you may want to try install these via MacPorts.
-
```
Comment (aartaka) [inline documents/README.org:209]: Why this newline removal? Any particular reason?

File: documents/README.org [hunk 2]
```diff
@@ -382,12 +380,20 @@ values can be configured in the respective init files.
...
+  - You need to have [[https://www.quicklisp.org/beta/][Quicklisp]] installed.
+    
+Steps:
 
-1. Add the following line to your Nyxt init file
+1. Add the following lines to your Nyxt init file. If your quicklisp directory differs, change it accordingly.
   #+begin_src lisp
-    (load-after-system :slynk "~/.config/nyxt/my-slynk.lisp")
+  (load (merge-pathnames #p"quicklisp/setup.lisp" (user-homedir-pathname)))
+  (ql:quickload 'slynk)
```
Comment (aartaka) [inline documents/README.org:391]: I suggest having two clauses here: a generic one and the one for Quicklisp users. This way we don't force everyone to use Quicklisp if they don't want to.

**Review by aartaka — COMMENTED** [review]
All good, just split SLY installation into Quicklisp and no-Quicklisp parts :)

File: documents/README.org (hunk 1 above)
Comment (opalvault) [reply documents/README.org:209]: Minor formatting fix. Line 209 created a gap between bulleted section headings when collapsed.

Comment (aartaka) [reply documents/README.org:209]: I guess it's more consistent this way :D

The next time, please, ensure that formatting changes are in a separate commit/PR from the content changes :)

File: documents/README.org (hunk 2 above)
Comment (opalvault) [reply documents/README.org]: This has been added, with a few extra clarifications, links, etc. Let me know if that looks alright :)

File: documents/README.org [hunk 3]
```diff
@@ -413,8 +415,51 @@ Steps:
...
+If you aren't using Quicklisp, you can configure Nyxt to start a Slynk server with the following steps.
+
+1. Ensure you have the Sly Package installed for Emacs with =package-install=. You may also clone the repository from [[https://github.com/joaotavora/sly.git][here]]. Should you choose to clone the repository, see the appropriate section in step 2.
+
+2. Add the following lines to your Nyxt's init.lisp (the configuration file) depending on whether you are using the Emacs Sly package, or you cloned the repository:
+   
+   - *If you are pointing Nyxt to your Emacs Sly package installation directory add these lines to your =nyxt/init.lisp=:*
+   #+NAME: nyxt/init.lisp
+   #+begin_src lisp
+   ;; Change the push directory below to where you've cloned Sly.
```
Comment (aartaka) [inline documents/README.org:430]: This comment should be in the next source listing, shouldn't it?

Comment (opalvault) [reply documents/README.org]: Good eye! Fixed. I think I've avoided any derps this time :P thanks!

File: documents/README.org (hunk 1 above)
Comment (opalvault) [reply documents/README.org:209]: I thought I replied to this, whoops. I will keep that in mind. Thanks for the heads up!

**Comment by aartaka** [comment]
Thanks for your help with documentation!

**Comment by Ambrevar** [comment]
There is also the `reset-asdf-registries` helper which can greatly simplify setp 2.  What do you think?

**Pull request merged**